src/PowerShellEditorServices/Services/PowerShellContext/PowerShellContextService.cs

```diff
@@ -1924,43 +1931,11 @@ internal void WriteOutput(string outputString, bool includeNewLine)
             }
         }
 
-        private void WriteExceptionToHost(Exception e)
+        private void WriteExceptionToHost(RuntimeException e)
         {
-            const string ExceptionFormat =
-                "{0}\r\n{1}\r\n    + CategoryInfo          : {2}\r\n    + FullyQualifiedErrorId : {3}";
-
-            if (!(e is IContainsErrorRecord containsErrorRecord) ||
-                containsErrorRecord.ErrorRecord == null)
-            {
-                this.WriteError(e.Message, null, 0, 0);
-                return;
-            }
-
-            ErrorRecord errorRecord = containsErrorRecord.ErrorRecord;
-            if (errorRecord.InvocationInfo == null)
-            {
-                this.WriteError(errorRecord.ToString(), String.Empty, 0, 0);
-                return;
-            }
-
-            string errorRecordString = errorRecord.ToString();
-            if ((errorRecord.InvocationInfo.PositionMessage != null) &&
-                errorRecordString.IndexOf(errorRecord.InvocationInfo.PositionMessage, StringComparison.Ordinal) != -1)
-            {
-                this.WriteError(errorRecordString);
-                return;
-            }
-
-            string message =
-                string.Format(
-                    CultureInfo.InvariantCulture,
-                    ExceptionFormat,
-                    errorRecord.ToString(),
-                    errorRecord.InvocationInfo.PositionMessage,
-                    errorRecord.CategoryInfo,
-                    errorRecord.FullyQualifiedErrorId);
-
-            this.WriteError(message);
+            var psObject = PSObject.AsPSObject(e.ErrorRecord);
+            s_writeStreamProperty.SetValue(psObject, s_errorStreamValue);
```

Comment:
It might also be possible to bake these two reflection values into a closed delegate, but it's not necessary

Reply:
talked offline... since the Enum type is not public, we'll defer this for now as it's more work than it's worth.